data/mods/Aftershock/items/weapons.json

```
@@ -268,7 +268,7 @@
     "type": "GUN",
     "symbol": "(",
     "color": "white",
-    "name": { "str": "aztlani bow" },
+    "name": { "str": "Aztlani bow" },
     "description": "An ornate bow used by Aztlani nobility during hunting rituals.  Designed to fell the plated creatures of Aztlan, its articulated and power assisted limbs allow it to fire projectiles with a strength comparable to modern assault rifles.  .\n\nWith its drawing aid system currently disabled, firing it will require truly superhuman strength.  It can be activated to toggle it drawing assist functionality.",
```

```suggestion
    "description": "An ornate bow used by Aztlani nobility during hunting rituals.  Designed to fell the plated creatures of Aztlan, its articulated and power assisted limbs allow it to fire projectiles with a strength comparable to modern assault rifles.  \n\nWith its drawing aid system currently disabled, firing it will require truly superhuman strength.  It can be activated to toggle it drawing assist functionality.",
```

Looks like this is a stray period?

---

Good catch, there's another one though, let me fix it too!